<!DOCTYPE HTML>
<!--
	Ion by TEMPLATED
	templated.co @templatedco
	Released for free under the Creative Commons Attribution 3.0 license (templated.co/license)
-->
<html>
	<head>
		<title>Research</title>
		<meta http-equiv="content-type" content="text/html; charset=utf-8" />
		<meta name="description" content="" />
		<meta name="keywords" content="" />
		<!--[if lte IE 8]><script src="js/html5shiv.js"></script><![endif]-->

			<link rel="stylesheet" href="css/style.css" />
		<script src="js/jquery.min.js"></script>
		<script src="js/skel.min.js"></script>
		<script src="js/skel-layers.min.js"></script>
		<script src="js/init.js"></script>
		<link rel="shortcut icon" href="images/ClevelandClinic.ico" type="image/vnd.microsoft.icon" />
		<!-- <link rel="shortcut icon" href="https://hms.harvard.edu/favicon.ico" type="image/vnd.microsoft.icon" /> -->
		<noscript>
			<link rel="stylesheet" href="css/skel.css" />
			<link rel="stylesheet" href="css/style.css" />
			<link rel="stylesheet" href="css/style-xlarge.css" />
		</noscript>
	</head>
	<body id="top">

		<!-- Header -->
			<header id="header" class="skel-layers-fixed">
				<script src="js/header.js" type="text/javascript"></script>
			</header>


		<!-- Main -->
			<section id="one" class="wrapper style1">

				<div class="container">

<div class="row oneandahalf">

<div class="10u">

<h2> Research </h2>

<!-- <p>We focus on the evolution of resistance in cancer and bacteria as a process - rather than any specific instance. By pulling our focus back from the details of a given disease, we hope that we can divine some of the evolutionary mechanisms driving resistance in ALL diseases.</p> -->

<p>
We now know that cancer proceeds via combined evolutionary and ecological processes. 
There is irrefutable evidence that populations of cancer cells change as individual cells mutate and selection forces act to shape the population into more fit phenotypes. 
We now understand that not only do individual cells escape these selection forces by interacting with their environment, including stromal cells like fibroblasts, but also with other tumor cells. 
However, while basic scientists have established these facts, little, if any, of this knowledge has translated into clinically actionable information; and, more frustratingly, while many scientists have begun to accept these facts, it has not changed the pursuit of ‘silver bullets’: drugs to act on single targets awash in a sea of heterogeneity. 
In Theory Division, we use a host of tools to approach this problem, with our focus on the evolutionary mechanisms themselves, rather than the specific solutions evolution finds. 
Below is a snapshot of a few of the things we like to think about...  please reach out with questions or ideas - science is a team sport!
</p>

<h3> Evolution on adaptive (fitness) landscapes </h3>

<p>
Evolution can be described in many ways. One mathematicall convenient way is consider evolution like a search for peaks (points of highest fitness) on a rugged landscape. 
If you allow each unique genotype to be a location in space (say the x,y coordinates on a map), then the phenotype is the 'height'. 
If you now consider each 'map' to be a different drug, you start to reason about how different therapies would affect populations under selection by them. 
We have modelled bacterial evolution in this way <a href="http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004493"> and found that evolution could be effectively 'steered'</a>, by a clever ordering of different landscapes, though in reality it is not perfectly controllable because of <a href="https://www.nature.com/articles/s41467-018-08098-6"> evolutionary contingencies</a>, but may be predictable. 
More recently, we have worked to incorporate machinery from statistical mechanics, together with the <a href="http://biotheory.phys.cwru.edu/people.html"> Hinczewski group in Case Physics</a> to formally derive <a href="https://www.biorxiv.org/content/10.1101/867143v1"> counter-diabatic driving protocols</a> to allow precise control of the speed and trajectory of populations as they navigate these landscapes. 

	 <br>
	 <table>

	<tr>
		<td>
			<figure align="center">
   <img src="images/research/dynamic_fitness.gif" alt=""  height="300px" width="auto">
  <figcaption>Olson R. and Østman B., "Using fitness landscapes to visualize
	 evolution in action."</figcaption>
</figure>


 </td>

 <td>
	 <iframe align="center" width="500" height="300" src="https://www.youtube.com/embed/P7Tb92qTmKw?autoplay=1&loop=1&playlist=P7Tb92qTmKw" frameborder="0"
	 allowfullscreen></iframe>
	   <figcaption>Here we show evolution proceeding on three different landscapes in a row. At time 20k and 40k the drugs (landscapes) are switched. In this case, we show an example where the middle drug, cefprozil, is used to steer the evolution of the third drug, ampicillin,
			  to a suboptimal peak. Details can be found <a href="http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004493">here</a>. </figcaption>
 </td>
</tr>
</table>

</p>


<h3> Evolutionary Bioreactor </h3>

<p>The emergence of drug resistance is the key stumbling block in our fight against cancer.
Though our tools can be extremely effective in the short term, it is inevitable that they willfail in the majority of cancers. 
As an evolutionary phenomenon, we know that the cancer we are treating will “figure out” a way to evade, subvert, and fight back against what we throw at it.
We know where the cancer will end up but have relatively little idea how it gets there.
We are designing and building a device based on the “morbidostat” framework from the study of antibiotic resistance which we call the <a href="https://www.biorxiv.org/content/10.1101/729434v2">EVE system (EVolutionary biorEactor)</a>.  
This will allow us to watch and measure cancer cells on their evolutionary journey towards resistance. 
It will subject the cells to the same therapies they would experience in patients, but allow us to observe how the cells grow and change much more closely than we ever could in people. 
The system will be fully robotically automated, providing us a tightly-controlled experimental environment where we can manipulate conditions and see how the cancer cells adjust. 
Information about the paths that cancers take to resistance can give us new insights about when and how we can intervene while we still can. 
Our device will allow us to ask and answer questions such as “Can we steer a cancer’s path to slow its progression to resistance?” and “Are there certain times that are better to act than others?”
	 <br>
	 <br>
	 <table>

	<tr>
		<td>
			<figure align="center"  >
   <img src="images/research/schematic.png" alt=""  height="300px" width="auto">
  <figcaption>Schematic of evolutionary bioreactor</figcaption>
</figure>


 </td>

 <td>
	 <figure align="center">
 <img src="images/research/morbidostat_sim.png" alt=""  height="300px" width="auto">
 <figcaption>Mathematical model of proposed dynamics</figcaption>
 </figure>
 </td>
</tr>
</table>

</p>



<h3> Evolutionary Game Theory </h3>

<p>We have an interest in the application of evolutionary game theory to the study of cancer, specifically to the evolution of resistance. 
Evolutionary game theory allows us to model and perturb the complex cooperative and competitive dynamics between sensitive tumor cells, resistant tumor cells, and the microenvironment. 
We use <a href="http://rsif.royalsocietypublishing.org/content/12/108/20150154.full">mathematical (systems of differential equations)</a> and <a href="https://www.nature.com/articles/bjc2011517">computational (numerical simulation)</a> models (right), and an <a href="https://www.nature.com/articles/s41559-018-0768-z">experimental game assay we have developed (co-culture and automated cell-counting)</a> (left) to investigate these interactions. 
	 <br>
	 <div align="center">
		 <img src="images/research/alectinib.png" alt=""  height="400px" style="margin-right: 1cm" width="auto">
	 <img src="images/research/simplex.gif" alt=""  height="400px" width="auto">
 </div>

</p>


<h3> Data Science </h3>

<p>With regards to data science, our interests span the breadth of clinical cancer research, ranging from developing novel biomarkers of disease, to selecting personalized therapies (including <a href="https://www.thelancet.com/journals/lanonc/article/PIIS1470-2045%2816%2930648-9/fulltext">personalized radiation therapy dosing</a>), and monitoring for resistance.  
Using datasets encompassing clinico-pathologic, biologic, and genomic data, we hope to design new tests, and lay the groundwork for novel anticancer therapies. 
To study large high dimensional datasets, we use a combination of high-performance computing and statistical approaches, on problems such as developing gene signatures for <a href="https://linkinghub.elsevier.com/retrieve/pii/S2589004220304806">drug sensitivity and resistance in Ewings Sarcoma</a>, understanding the functional role of newly characterized <a href="https://www.nature.com/articles/s41467-018-07657-1">non-coding RNA</a>, and determining methods of <a href="https://www.biorxiv.org/content/10.1101/854695v2">combining microRNAs into therapeutic cocktails</a>.  
	 <br>
	 <div align="center">
	 <img src="images/research/convergence_sig.jpg" alt=""  height="400px" width="auto">
 </div>

</p>

<h3>Lab Research Statistics</h3>

Lab h-index = 25

<table style="width:100%" bgcolor="#FFFFFF">
	<tr>
		<td style="width:49%">
			<img src="images/labstats/CiteData.png" width="99%"/>
		</td>
		<td style="width:49%">
			<img src="images/labstats/PubData.png" width="99%"/>
		</td>
	</tr>
</table>

	</div>
</div>
</div>

			</section>

		<!-- Footer -->
			<footer id="footer">
				<script src="js/footer.js" type="text/javascript"></script>
				<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-112256444-1', 'auto');
  ga('send', 'pageview');

</script>
			</footer>

	</body>
</html>
