<!DOCTYPE HTML>
<!--
	Ion by TEMPLATED
	templated.co @templatedco
	Released for free under the Creative Commons Attribution 3.0 license (templated.co/license)
-->
<html>
	<head>
		<title>Research</title>
		<meta http-equiv="content-type" content="text/html; charset=utf-8" />
		<meta name="description" content="" />
		<meta name="keywords" content="" />
		<!--[if lte IE 8]><script src="js/html5shiv.js"></script><![endif]-->
		<link rel="stylesheet" href="css/style.css" />
		<script src="js/jquery.min.js"></script>
		<script src="js/skel.min.js"></script>
		<script src="js/skel-layers.min.js"></script>
		<script src="js/init.js"></script>
		<link rel="shortcut icon" href="images/ClevelandClinic.ico" type="image/vnd.microsoft.icon" />
		<!-- <link rel="shortcut icon" href="https://hms.harvard.edu/favicon.ico" type="image/vnd.microsoft.icon" /> -->
		<noscript>
			<link rel="stylesheet" href="css/skel.css" />
			<link rel="stylesheet" href="css/style.css" />
			<link rel="stylesheet" href="css/style-xlarge.css" />
		</noscript>
	</head>
	<body id="top">
		<!-- Header -->
		<header id="header" class="skel-layers-fixed">
			<script src="js/header.js" type="text/javascript"></script>
		</header>

		<!-- Main -->
		<section id="one" class="wrapper style1">
			<div class="container">
				<div class="row oneandahalf">
				
					<!-- egtplot -->
					<div class="10u">
						<h3>egtplot: A Python Package for 3-Strategy Evolutionary Games</h3>
						<a href="http://joss.theoj.org/papers/10.21105/joss.00735"><img src="http://joss.theoj.org/papers/10.21105/joss.00735/status.svg"></a>
						<a href="https://www.biorxiv.org/content/10.1101/300004v2"><img src="https://img.shields.io/badge/bioRxiv-10.1101/300004-red"></a>
						<a href="https://github.com/mirzaevinom/egtplot"><img src="https://img.shields.io/badge/GitHub-mirzaevinom/egtplot-blue.svg"></a>
						<br>
						<a href="https://pepy.tech/project/egtplot"><img src="https://pepy.tech/badge/egtplot"></a>
						<a href="https://pypi.org/project/egtplot/"><img src="https://badge.fury.io/py/egtplot.svg"></a>
						<a href="https://opensource.org/licenses/MIT"><img src="https://img.shields.io/badge/License-MIT-yellow.svg"></a>
						<p>
							Evolutionary game theory is a very broad modeling framework that effectively describes many aspects
							of biological cooperation and competition.
							Visualization of three-strategy evolutionary games has historically been difficult within the
							Python ecosystem.
							We have created a package to ease visualization efforts that is capable of displaying both static
							and animated dynamics with the game space.
							For detailed software usage instructions we refer readers to our interactive
							<a href="https://github.com/mirzaevinom/egtplot/blob/master/egtplot_demonstration.ipynb ">jupyter 
							notebook</a>.
							More details are available in publication in
							<a href="https://joss.theoj.org/papers/0cc2d656d8a02a84e0cde905207820d7">Journal of Open Source 
							Software</a> where you can also find installation instructions.
							<br>
							<br>
							<div align="center">
								<img src="images/software/egtplot_animation.gif" alt=""  height="400" width="40%">
								<img src="images/software/egtplot_simplex.png" alt=""  height="400" width="40%">
							</div>
						</p>
					</div>
					
					<!-- sigQC -->
					<div class="10u">
						<h3>sigQC - A new standard for gene signature quality control </h3>
						<a href="https://www.nature.com/articles/s41596-019-0136-8"><img src="https://img.shields.io/badge/Nat%20Protoc-10.1038/s41596--019--0136--8-red.svg"></a>
						<a href="https://www.biorxiv.org/content/10.1101/203729v2"><img src="https://img.shields.io/badge/bioRxiv-10.1101/203729-red"></a>
						<a href="https://github.com/andrewdhawan/sigQC"><img src="https://img.shields.io/badge/GitHub-andrewdhawan/sigQC-blue.svg"></a>
						<br>
						<a href="https://doi.org/10.5281/zenodo.1319848"><img src="https://zenodo.org/badge/DOI/10.5281/zenodo.1319848.svg"></a>
						<a href="https://CRAN.R-project.org/package=sigQC"><img src="http://cranlogs.r-pkg.org/badges/grand-total/sigQC"></a>
						<a href="https://CRAN.R-project.org/package=sigQC"><img src="http://www.r-pkg.org/badges/version/sigQC"></a>
						<a href="https://www.gnu.org/licenses/gpl-3.0"><img src="https://img.shields.io/badge/License-GPL%20v3-green.svg"></a>
						<p>
							sigQC is an R package, available on CRAN, that we have worked in collaboration to develop,
							defining an integrated methodology for gene signature quality control.
							Increasing amounts of genomic data mean	that gene expression signatures are becoming critically
							important tools, poised to make a large impact on the diagnosis, management and prognosis for
							a number of diseases.
							For the purposes of this package, we define the term gene signature to mean: ‘a set of genes
							whose co-ordinated mRNA expression pattern is representative of a biological pathway, process,
							or phenotype, or clinical outcome.’
							<br>
							A key issue with gene signatures of this nature is whether the expression of many genes can be
							summarised as a single score, or whether multiple components are represented.
							In this package, we have automated the testing of a number of quality control metrics designed
							to test whether a single score, such as the median or mean, is an appropriate summary for the
							genes’ expression in a dataset. 
							The tools in this package enable the visualization of properties of a set of genes in a specific
							dataset, such as expression profile, variability, correlation, and comparison of methods of 
							standardisation and scoring metrics.
							<br>
							<br>
							Read more about it on Andrew’s <a href="https://andrewdhawan.wordpress.com/2017/10/16/sigqc-a-new-standard-for-gene-signature-quality/">
							blog</a>,
							or in the <a href="https://www.nature.com/articles/s41596-019-0136-8">Nature Protocols manuscript</a>.
							<br>
							Try it for yourself today as well --
							<a href="https://cran.r-project.org/web/packages/sigQC/ ">CRAN package</a>.
							<br>
							<div align="center">
								<img src="images/software/sigQC_flowchart.jpg" alt=""  height="400px" width="auto">
							</div>
						</p>
					</div>
				
					<!-- TDAstats -->
					<div class="10u">
						<h3>TDAstats: R pipeline for computing persistent homology in topological data analysis</h3>
						<a href="https://doi.org/10.21105/joss.00860"><img src="http://joss.theoj.org/papers/10.21105/joss.00860/status.svg"></a>
						<a href="https://github.com/rrrlw/TDAstats"><img src="https://img.shields.io/badge/GitHub-rrrlw/TDAstats-blue.svg"></a>
						<a href="https://zenodo.org/badge/latestdoi/130141540"><img src="https://zenodo.org/badge/130141540.svg"></a>
						<br>
						<a href="https://CRAN.R-project.org/package=TDAstats"><img src="http://cranlogs.r-pkg.org/badges/grand-total/TDAstats"></a>
						<a href="https://CRAN.R-project.org/package=TDAstats"><img src="http://www.r-pkg.org/badges/version/TDAstats"></a>
						<a href="https://www.gnu.org/licenses/gpl-3.0"><img src="https://img.shields.io/badge/License-GPL%20v3-blue.svg"></a>
						<br>
						<a href="https://travis-ci.org/rrrlw/TDAstats"><img src="https://travis-ci.org/rrrlw/TDAstats.svg?branch=master"></a>
						<a href="https://ci.appveyor.com/project/rrrlw/TDAstats"><img src="https://ci.appveyor.com/api/projects/status/github/rrrlw/TDAstats?branch=master&svg=true"></a>
						<a href="https://codecov.io/github/rrrlw/TDAstats?branch=master"><img src="https://img.shields.io/codecov/c/github/rrrlw/TDAstats/master.svg"></a>
						<p>
							The TDAstats package is a comprehensive pipeline for conducting topological data analysis in R, 
							allowing useRs to calculate, visualize, and conduct statistical inference on persistent homology
							in a Vietoris-Rips simplicial complex.
							The increased use of next-generation sequencing (and other newer experimental methods) has
							resulted in the increased prevalence of high-dimensional datasets.
							Although dimension reduction methods (e.g. principal component analysis) have been used with
							some success, it would be ideal to preserve all of the high-dimensional information within a
							dataset during data analysis.
							This is where topological data analysis and persistent homology come in.
							<br>
							Currently, the fastest method to compute persistent homology is the
							<a href="https://github.com/Ripser/ripser">Ripser C++ library</a>, a variant of which is wrapped
							by TDAstats using <a href="https://CRAN.R-project.org/package=Rcpp">Rcpp</a>.
							This allows TDAstats to outpace other comprehensive pipelines for topological data analysis while
							also being implemented in R, a language familiar to many data scientists.
							TDAstats allows visualization of persistent homology with either topological barcodes (below,
							left panel) or persistence diagrams (below, right panel) as publication-quality figures using the
							<a href="https://CRAN.R-project.org/package=ggplot2">ggplot2</a> library, allowing useRs to fully 
							customize the plots.
							Lastly, TDAstats is the first software library (as far as we know) that allows nonparametric 
							statistical inference on persistent homology using permutation tests.
							This allows useRs to compare the topology, or "shape", of two datasets and determine if they originate 
							from similar populations.
							Current efforts are being directed at applying TDAstats to elucidate knowledge about topological
							graph theory.
							<br>
							TDAstats has been published in the
							<a href="https://joss.theoj.org/papers/0ac4998ce691c718c4518201b86ef87d">Journal of Open Source 
							Software</a>.
							For more details, you can visit its <a href="https://CRAN.R-project.org/package=TDAstats">CRAN
							page</a>; the associated <a href="https://github.com/rrrlw/TDAstats">GitHub repo</a> contains the
							code behind TDAstats.
							See the <a href="https://github.com/rrrlw/TDAstats/blob/master/README.md">README.md</a> file for 
							information on how to get started.
							<br>
							<div align="center">
								<img src="images/software/TDAstats.png" alt="" height="300px" width="auto">
							</div>
						</p>
					</div>

					<!-- TDAstats -->
					<div class="10u">
						<h3>EVE: The EVolutionary biorEactor</h3>
						<p>
						The EVolutionary biorEactor is a tool to study the evolution of drug resistance in bacteria.
						The paths that identical organisms take to achieve resistance to the same drug differs due to the inherent randomness of evolution.
						By monitoring the phenotypes (and potentially also genotypes) of multiple cultures grown in similar conditions, we can get an idea of which evolutionary paths occur most frequently and target drugs consequently.	

						Software is written in Python and interfaces with circuit board through I2C. Innovative multi-threading is used to manage multiple culture units simulataneously. 
						A Tornado-based web server is used to configue, start, and monitor experiments.

							<br>
							<div align="center">
								<img src="images/software/eve-sw.png" alt="" height="300px" width="auto">
							</div>
						</p>
					</div>

				</div>
			</div>
			
		</section>

		<!-- Footer -->
		<footer id="footer">
			<script src="js/footer.js" type="text/javascript"></script>
			<script>
				(
					function(i,s,o,g,r,a,m)
					{
						i['GoogleAnalyticsObject']=r;
						i[r]=i[r]||function()
						{
							(i[r].q=i[r].q||[]).push(arguments)
						},i[r].l=1*new Date();
						a=s.createElement(o),
						m=s.getElementsByTagName(o)[0];
						a.async=1;
						a.src=g;
						m.parentNode.insertBefore(a,m)
					}
				)(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

				ga('create', 'UA-112256444-1', 'auto');
				ga('send', 'pageview');

			</script>
		</footer>
	</body>
</html>
